dbms/src/Common/TiFlashMetrics.h

```diff
@@ -764,11 +775,12 @@ namespace DB
 
 
 /// Buckets with boundaries [start * base^0, start * base^1, ..., start * base^(size-1)]
+template <size_t size>
```

Comment:
IMO, It is better to apply the check this way: https://github.com/CalvinNeo/tics/pull/5. It keeps most of the lines unchanged. If someone add metrics with wrong param mistakenly, it can not pass the CI testing.